flottform/forms/src/internal.ts

```diff
@@ -51,13 +51,15 @@ export type FlottformState =
 	| 'done'
 	| 'error';
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
 export type Logger = {
 	debug: (...args: any[]) => void;
 	info: (...args: any[]) => void;
 	log: (...args: any[]) => void;
 	warn: (...args: any[]) => void;
 	error: (...args: any[]) => void;
 };
+/* eslint-enable @typescript-eslint/no-explicit-any */
```

Comment:
I think it's ok like this, but what do you think about `debug: typeof console.debug` etc? Would that work as well?

Reply:
Yes, this also works. I changed it in new commit